src/Lombiq.OSOCE.Web/Program.cs

```diff
@@ -14,7 +15,12 @@
 // create UI tests check out the project.
 builder.Services
     .AddSingleton(configuration)
-    .AddOrchardCms();
+    .AddOrchardCms(orchardCoreBuilder =>
+        orchardCoreBuilder.ConfigureFeaturesGuard(
+            new Dictionary<string, IEnumerable<string>>
+            {
+                ["OrchardCore.Twitter"] = new List<string> { "Lombiq.UIKit", "Lombiq.ChartJs" },
```

Comment:
Same for the NuGet solution.

```suggestion
                ["OrchardCore.Twitter"] = new[] { "Lombiq.UIKit", "Lombiq.ChartJs" },
```